src/beast/beast/container/detail/aged_ordered_container.h

```diff
@@ -1331,6 +1340,7 @@ template <bool IsMulti, bool IsMap, class Key, class T,
 aged_ordered_container <IsMulti, IsMap, Key, T, Clock, Compare, Allocator>::
 aged_ordered_container (aged_ordered_container const& other)
     : m_config (other.m_config)
+    , m_cont (other.m_cont.comp())
```

Comment:
The constructors that take an `other` and `Allocator` param do not copy `other.m_cont.comp()` as we do here. Why is that? (it looks like it needs it to me, but it also looks like you intentionally left them out).